sources/osgUtil/NodeGizmo.js

```diff
@@ -535,10 +535,11 @@ NodeGizmo.prototype = MACROUTILS.objectInherit( MatrixTransform.prototype, {
 
     getTransformType: function ( node ) {
         var n = node;
-        while ( n.parents.length > 0 ) {
+        var parents = n.getParents();
```

Comment:
Infinite loop... should be ``while(n.getParents().length)`` and ``n = n.getParents()[0]``